python/lsst/sitcom/vandv/hexapod.py

```diff
@@ -0,0 +1,148 @@
+import asyncio
+
+import numpy as np
+import os
+import pandas as pd
+import yaml
+
+
+__all__ = [
+    "check_hexapod_lut",
+    "get_hexapod_configuration",
+    "print_hexapod_compensation_values",
+    "print_hexapod_uncompensation_values",
+]
+
+
+async def check_hexapod_lut(component, timeout=10.0):
+    """Get the compensation offsets and print them. This is a way of telling
+    if the component (hexapod) has enough input to do look-up table (LUT)
+    compensation or not.
+
+    Parameters
+    ----------
+    component : str
+        Name of the component: mthexapod_1 for Camhex or mthexapod_2 for M2Hex
+    timeout : float
+
+    Note
+    ----
+    The target events are what the hexa CSC checks. If one is missing,
+    the entire LUT will not be applied. It also needs to see an
+    `uncompensatedPosition` (a move would trigger that) in order to
```

Comment:
In the test that I run in the TTS, it never saw an uncompensatedPosition. Is there anything am I missing?